### PR TITLE
Add new mental health question

### PR DESCRIPTION
**Pull request by patrickjfl**
# Context

https://trello.com/c/wQ2nvVFN/664-health-new-question-are-they-engaged-with-any-mental-health-services-in-custody

E2E PR: https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-e2e/pull/62

# Changes in this PR

Adds new question to the mental health form page.

### Before

Screenshot 2024-01-17 at 09 20 13

### After
Screenshot 2024-01-17 at 09 19 48

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Are any changes required to the e2e tests?
- [ ] If you've added a new route, have you added a new
  `auditEvent`? (see `server/routes/apply.ts` for examples)
- [ ] Are there environment variables or other infrastructure configuration which needs to be included in this release?
- [ ] Are there any data migrations required. Automatic or manual?
- [ ] Does this rely on changes being deployed to the CAS API?

## Post merge checklist

Once we've merged it will be auto-deployed to the dev environment.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui).

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

File: server/form-pages/utils/questions.ts [hunk 1]
```diff
@@ -523,6 +523,10 @@ export const getQuestions = (name: string) => {
           answers: yesOrNo,
         },
         servicesDetail: { question: 'Please state which services.' },
+        isEngagedWithServicesInCustody: {
```
Comment (libuk) [inline server/form-pages/utils/questions.ts:526]: Should we also add to the test fixtures `applicationData.json` and `applicationDocument.json`?

**Review by libuk — APPROVED** [review]
Looks good!

File: server/form-pages/utils/questions.ts (hunk 1 above)
Comment (patrickjfl) [reply server/form-pages/utils/questions.ts:526]: oh yes, missed this! thanks

**Review by libuk — APPROVED** [review]
Nice one!

**Pull request merged**